Generals/Code/GameEngine/Source/GameClient/GameClient.cpp

```diff
@@ -457,6 +459,9 @@ void GameClient::reset( void )
 	// clear any drawable TOC we might have
 	m_drawableTOC.clear();
 
+	// TheSuperHackers @fix Mauller nextDrawableID was never being reset.
```

Comment:
`// TheSuperHackers @fix Mauller 13/04/2025 Resets drawable id so it does not keep growing over the lifetime of a game session.`